packages/cli/src/telemetry/cliTelemetryEvents.ts

```diff
@@ -45,6 +45,7 @@ export enum TelemetryEvent {
 
   DeployStart = "deploy-start",
   Deploy = "deploy",
+  DeployAad = "deployAad",
```

Comment:
Note: Please make sure correctly tag the property in cli telemetry.

Reply:
sure, will check it after rc.